src/main/java/org/elasticsearch/index/query/IdsQueryBuilder.java

```diff
@@ -83,7 +77,7 @@ public IdsQueryBuilder ids(String... ids) {
     }
 
     /**
-     * Gets the ids for the query.
+     * Returns the ids for the query.
      */
     public Collection<String> ids() {
```

Comment:
I know I introduced this, but re-reading this getter again I wonder if we should return a Set here if we know its a Set. Just to be more explicit.